src/media/css/app-list.styl

```diff
@@ -110,16 +110,31 @@ section.langpacks > div {
     }
     .app-list-app {
         border-bottom: $border;
-        border-right: $border;
         padding: $desktop-tile-padding;
         width: 50%;
 
+        +ltr() {
+            border-right: $border;
```

Comment:
I don't see why this is in a `+ltr()` block if it gets overriden in the RTL case.